index.html

```diff
@@ -114,13 +124,143 @@ <h2>Introduction</h2>
+      <li>MUST NOT include any vocabulary, syntax or attribute value associated with a Feature whose disposition is prohibited in the profile.</li>
+    </ul>
+    <p class="note">
+      A <a>Document Instance</a>, by definition, satisfies the requirements of Section 3.1 at [[TTML2]], and hence a Document Instance that conforms to a profile defined herein is also a conforming TTML2 Document Instance.
+    </p>
+    <p>
+      A <a>presentation processor</a> that conforms to the profile defined in this specification:
+    </p>
+    <ul style="list-style-type:disc">
+      <li>MUST satisfy the Generic Processor Conformance requirements at Section 3.2.1 of [[TTML2]]</li>
+      <li>MUST satisfy all normative provisions specified by the profile; and</li>
+      <li>MUST implement presentation semantic support for every Feature designated as permitted or required by the profile, subject to any additional constraints on each Feature as specified by the profile.</li>
+      <li>MAY implement presentation semantic support for every Feature designated as optional by the profile, subject to any additional constraints on each Feature as specified by the profile.</li>
+    </ul>
+    <p>
+      A <a>transformation processor</a> that conforms to the profile defined in this specification:
+    </p>
+    <ul style="list-style-type:disc">
+      <li>MUST satisfy the Generic Processor Conformance requirements at Section 3.2.1 of [[TTML2]];</li>
+      <li>MUST satisfy all normative provisions specified by the profile; and</li>
+      <li>MUST implement transformation semantic support for every Feature designated as permitted or required by the profile, subject to any additional constraints on each Feature as specified by the profile.</li>
+      <li>MAY implement transformation semantic support for every Feature designated as optional by the profile, subject to any additional constraints on each Feature as specified by the profile.</li>
+    </ul>
+    <p class="note">
+      The use of the terms <a>presentation processor</a> 
+      and <a>transformation processor</a> within this document does not imply conformance <i>per se</i> to any of the Standard Profiles defined in [[TTML2]].
+      In other words, it is not considered an error for a <a>presentation processor</a> or <a>transformation processor</a> to conform to the profile defined in this document without also conforming to the TTML2  Presentation Profile or the TTML2 Transformation Profile.
+    </p>
+    <p class="note">
+      This document does not specify presentation processor or transformation processor behavior when processing or transforming a non-conformant Document Instance.
+    </p>
+    <p class="note">
+      The permitted and prohibited dispositions do not refer to the specification of a ttp:feature or ttp:extension element as being permitted or prohibited within a ttp:profile element.
+    </p>
```

Comment:
Is this note really needed? Can avoid having the reader/implementer worry about `ttp:feature` et al.

Reply:
I think this is an example of retaining lessons learned in the past. I'd keep it. It avoids a potential ambiguity.